EMML/1-1000/EMML52.xml

```diff
@@ -99,6 +100,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
       <revisionDesc>
          <change who="PL" when="2017-11-30">Created minimal record with link from data sent from Daniel Gullo, HMML
                             project.</change>
+                            <change when="2022-04-05" who="DB">changes on dimantions</change>
```

Comment:
just correct the spelling 
changes on **dimensions**
otherwise all is perfect, you rock it girl!!!